EIPS/eip-7923.md

```diff
@@ -282,6 +329,19 @@ index 2ccfd090..5950a4d4 100644
 -        buf = memoryview(self._bytes)
 +        buf = memoryview(self.memview)
          buf[destination : destination + length] = buf[source : source + length]
+diff --git a/eth/vm/transaction_context.py b/eth/vm/transaction_context.py
+index 79b570e9..5943f897 100644
+--- a/eth/vm/transaction_context.py
++++ b/eth/vm/transaction_context.py
+@@ -36,6 +36,9 @@ class BaseTransactionContext(TransactionContextAPI):
+         # post-cancun
+         self._blob_versioned_hashes = blob_versioned_hashes or []
+
++        # eip-7923
++        self.num_pages = 0
```

Comment:
I'm not sure what this `transaction_context.py` file does, but are these properties allowed to be edited? In our JS codebase we do not edit properties of txs, we have to explicitly create new txs in order to change values. This `num_pages` definitely changes. I'm asking because if a tx runs (for instance due to an RPC call, or just normal execution) and somehow it runs again, then I think this `num_pages` is thus set as nonzero where it should be zero? Is this the right place for this param?

Reply:
the transaction context keeps the log counter, which is also mutable... the other reasonable place to put it would be in the State object.

Reply:
Ok, if it's mutable then the location is fine :+1: :smile: